shared_model/builders/protobuf/builder_templates/query_template.hpp

```diff
@@ -182,14 +182,12 @@ namespace shared_model {
       }
 
       auto getAccountDetail(
-          const size_t page_size,
+          size_t page_size,
```

Comment:
It is not a problem, but here it can remain const and this will be meaningful (because the method is not only declared but is defined here).

can be const, because the method's body does not modify the variable.